app/assets/javascripts/feedback_form.js

```diff
@@ -71,7 +71,7 @@ $(document).on('turbolinks:load', function(){
         var flashHtml = "<div class='alert alert-" + val[0] + "'><button type='button' class='close' data-dismiss='alert' aria-hidden='true'>&times;</button>" + val[1] + "</div>";
 
         // Show the flash message
-        $('div.flash_messages').html(flashHtml);
+        $('div.flashes').html(flashHtml);
```

Comment:
The flash message still isn't loading for me unless I reload the page, which may be why the test still isn't working?

Reply:
yeah, turned out I needed to test it in incognito to avoid stale messages appearing. there was more that needed fixing in the javascript! thx for checking